### ISSUE TITLE
Getting path errors if CLI command is executed from the same directory as the config and input nb

### ISSUE THREAD

**Issue by willingc** [muellerzr/nbquarto#5]
Hi @muellerzr,

Loving this so far. Thank you!

I'm seeing this error which I can look at closer on the weekend. Either I missed that you can't execute the process from the same directory as the `config.yaml` or there is a small logic bug. Executes well when I go up one directory level.

```
λ › nbquarto-process \                                                                                                        ~/nbs
--config_file config.yaml \                               
--notebook_file penguins.ipynb \                         
--output_folder output/
Traceback (most recent call last):
  File "/Users/willingc/.pyenv/versions/3.11.2/bin/nbquarto-process", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/willingc/nbquarto/src/nbquarto/cli.py", line 156, in main
    process_notebook(args.notebook_file, args.config_file, args.output_folder)
  File "/Users/willingc/nbquarto/src/nbquarto/cli.py", line 82, in process_notebook
    output_location = output_folder / notebook_location.relative_to(documentation_source)
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/willingc/.pyenv/versions/3.11.2/lib/python3.11/pathlib.py", line 730, in relative_to
    raise ValueError("{!r} is not in the subpath of {!r}"
ValueError: 'penguins.ipynb' is not in the subpath of 'nbs' OR one path is relative and the other is absolute.
λ › nbquarto-process \                                                                                                        ~/nbs
--config_file config.yaml \
--notebook_file ./penguins.ipynb \
--output_folder output/
Traceback (most recent call last):
  File "/Users/willingc/.pyenv/versions/3.11.2/bin/nbquarto-process", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/willingc/nbquarto/src/nbquarto/cli.py", line 156, in main
    process_notebook(args.notebook_file, args.config_file, args.output_folder)
  File "/Users/willingc/nbquarto/src/nbquarto/cli.py", line 82, in process_notebook
    output_location = output_folder / notebook_location.relative_to(documentation_source)
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/willingc/.pyenv/versions/3.11.2/lib/python3.11/pathlib.py", line 730, in relative_to
    raise ValueError("{!r} is not in the subpath of {!r}"
ValueError: 'penguins.ipynb' is not in the subpath of 'nbs' OR one path is relative and the other is absolute.
λ › cd ..                                                                                                                     ~/nbs
λ › nbquarto-process \                                                                                                            ~
--config_file nbs/config.yaml \
--notebook_file nbs/penguins.ipynb \
--output_folder nbs/output/
INFO:nbquarto.cli:Successfully processed notebook at nbs/penguins.ipynb and saved to nbs/output/penguins.qmd
λ ›                        
```

**Comment by muellerzr** [comment]
What's your directory structure like? :)

**Comment by willingc** [comment]
I tried a few combos:

1. Path error if CLI command given in `nbs` directory and separate directories for notebooks input and output artifacts
```
nbs
--- config.yaml
--- inputs
          penguins.ipynb
--- outputs
```
2. Both `config.yaml` and source notebook in same level of `nbs` folder. Executing command in `nbs` folder.
```
nbs
--- config.yaml
--- penguins.ipynb
--- outputs
```
3. Backing out 1 level higher than the `nbs` folder and executing the command worked perfectly.

**Comment by muellerzr** [comment]
What's your config.yaml like? Once I have all that, I can give you a full complete answer on the behavior :)

**Comment by muellerzr** [comment]
I *think* it's a relative pathing issue, but I'd need to know what you're setting in the config.yaml to be sure

**Comment by willingc** [comment]
OK I'm off. I did have the wrong path in the config.yaml. You were correct on the relative path issue. Thanks and sorry for the noise.

**Comment by muellerzr** [comment]
Np! Definitely not noise, means I need a better error for that happening, let's keep this issue opened until I'm content :)

**Comment by muellerzr** [comment]
Hi @willingc, turns out it is an interesting behavior with pathlib! With the new fix of #7, all of what you tried to do *should* work now :)
